core/src/tx/mod.rs

```diff
@@ -49,26 +49,14 @@ pub struct ValidationDetails<'a> {
 }
 
 /// A transaction payload containing some generic `CallData`.
-#[derive(Derivative)]
-#[derivative(
-    Clone(bound = "CallData: Clone"),
-    Debug(bound = "CallData: std::fmt::Debug"),
-    Eq(bound = "CallData: std::cmp::Eq"),
-    Ord(bound = "CallData: std::cmp::Ord"),
-    PartialEq(bound = "CallData: std::cmp::PartialEq"),
-    PartialOrd(bound = "CallData: std::cmp::PartialOrd")
-)]
+#[derive(Clone, Debug)]
 pub struct Payload<CallData> {
     pallet_name: Cow<'static, str>,
     call_name: Cow<'static, str>,
     call_data: CallData,
     validation_hash: Option<[u8; 32]>,
 }
 
-/// A boxed transaction payload.
-// Dev Note: Arc used to enable easy cloning (given that we can't have dyn Clone).
-pub type BoxedPayload = Payload<Arc<dyn EncodeAsFields + Send + Sync + 'static>>;
```

Comment:
this doesn't work anymore?

I recall James talking about this as well at some point

Reply:
Yeah, We removed this because `EncodeAsFields` is no longer object-safe with the new `scale-type-resolver` integration. So we could add an `EncodeAsFieldsWithResolver<R>` trait to `scale_encode` to make it work. But we decided against it for now, because we asked in the public element room if anyone is actually using this and noone was apposed to it being removed.